dashboard/src/actions/repos.ts

```diff
@@ -165,8 +167,18 @@ export const installRepo = (
           secrets["ca.crt"] = btoa(customCA);
         }
       }
+      let syncJobPodTemplateObj = {};
+      if (syncJobPodTemplate.length) {
+        syncJobPodTemplateObj = yaml.load(syncJobPodTemplate);
```

Comment:
This needs to be `yaml.safeLoad()`. I fixed this in #1118 with a test to show why :)